var/spack/repos/jcsda-emc/packages/crtm-fix/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
Just wondering, has this been added to the bundles that need it?

Reply:
No. I really don't know in what context these files are used besides GSI. Should it just be anything that uses CRTM? I think it should be a variant like `depends_on(crtm-fix, when='+fix', in whatever bundle it goes in.

Reply:
Understood. If the package isn't needed except for GSI, then we can do as you suggest, create a variant and have it off by default.